voxelSize: "0.01"
sacDistanceThreshold: "0.02"
sacMaxIterations: "1000"
minClusterSize: "10"
maxClusterSize: "25000"
clusterTolerance: "0.02"
boundingBoxVolumefeatureLowerLimit: "0.0001"
boundingBoxVolumefeatureUpperLimit: "0.0008"
